clients/linux/src/backend.rs

```diff
@@ -220,14 +218,14 @@ impl LbCore {
     pub fn open(&self, id: &Uuid) -> Result<(FileMetadata, String), String> {
         match self.file_by_id(*id) {
             Ok(meta) => match self.read(meta.id) {
-                Ok(decrypted) => Ok((meta, decrypted.secret)),
+                Ok(decrypted) => Ok((meta, String::from_utf8(decrypted).unwrap())),
```

Comment:
@steverusso FYI documents are not necessarily UTF-8 because we support binary, use `String::from_utf8_lossy` to fill in invalid UTF characters with those weird question marks in boxes or whatever and not crash

Reply:
Ah ok, thanks. Fixed :+1: